### ISSUE TITLE
i can see only white screen after updating gutenberg 6.1

### ISSUE THREAD

**Issue by MYIG** [WordPress/gutenberg#16502]
I can see white screen only after updating it. I restore it to 6.0 and it works well now... I don't know why...

**Comment by aduth** [comment]
To help debug, could you provide:

- What browser are you using?
- Can you share any other plugins on your site which might interact with the Gutenberg editor?
- If able, can you provide any messages which are shown in the browser's Developer Tools Console ([instructions, typically <kbd>F12</kbd>](https://javascript.info/devtools))

**Comment by MYIG** [comment]
i am using vivaldi browser now.
i have many block plugin but i don't activate them all. and... jetpack, rank math, easy social share buttons, related(its name is 'related'), KaTeX, LuckyWP Table of Contents, Oxygen, Visual Link Preview
how to tell you message? just F12?

**Comment by aduth** [comment]
Do you have another browser on your computer that you could test with, to see if you have the same issue?

Technically, Vivaldi is not a [supported browser](https://make.wordpress.org/core/handbook/best-practices/browser-support/). That said, if we can try to pinpoint a specific difference between 6.0 and 6.1 versions of the plugin, it would be good to try to resolve this.

**Comment by MYIG** [comment]
um... I don't test with other browsers. but it is quite a tiresome;; I should test it then restore my plugins again and again?

**Comment by quicoto** [comment]
I've opened a new issue before finding this one.

I've posted a screenshot of the console error and more details

https://github.com/WordPress/gutenberg/issues/16515

**Comment by aduth** [comment]
I'm closing #16502 and #16517 in favor of this one, to simplify tracking.

As much detail as can be provided will assist in debugging.

There's a potential incompatibility which may explain at least part of the issue here, slated for fix in #16518 .

The error in the screenshot of #16515 may be of a different cause, as it stems from the `@wordpress/compose` module (most recently updated in #16065, cc @youknowriad ).

**Comment by aduth** [comment]
Related: https://meta.trac.wordpress.org/ticket/4597

**Comment by epiqueras** [comment]
>The error in the screenshot of #16515 may be of a different cause, as it stems from the @wordpress/compose module (most recently updated in #16065, cc @youknowriad ).

`compose` was part of the stack trace when I found the error that I fixed in #16518 so I think these are the same issues exposed by different browser dev tools.

**Comment by trudslev** [comment]
I have this problem and if I deactivate Jetpack the problem disappears.

**Comment by abwdvm** [comment]
We have this problem... I deactivated Gutenberg and the problem disappears.
Windows, Chrome everything up to date....

**Comment by epiqueras** [comment]
Fixed in #16518 .

**Comment by talldan** [comment]
Thanks for reporting this @MYIG, I'm closing as it should be fixed by #16518.
